### PR TITLE
Refactor logging statements in common,e2e test files

### PR DESCRIPTION
**Pull request by omerap12**
#### What type of PR is this?
Transition to structural logging in compliance with [Kubernetes logging guidelines](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/logging.md).

/kind cleanup
#### Which issue(s) this PR fixes:

As part of #6905, I'm splitting this into multiple PRs rather than submitting a single large PR.
#### Special notes for your reviewer:
I didn't change `klog.Fatalf` because I'm uncertain if or how it should be modified.
#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs
NONE
```

### CODE REVIEW

**Review by jackfrancis — APPROVED** [review]
/lgtm

**Comment by adrianmoisey** [comment]
Hey @voelzmo, can you approve this?
/assign voelzmo

File: vertical-pod-autoscaler/e2e/v1/admission_controller.go [hunk 1]
```diff
@@ -157,7 +157,7 @@ var _ = AdmissionControllerE2eDescribe("Admission-controller", func() {
 			apiv1.ResourceCPU:    ParseQuantityOrDie("100m"),
 			apiv1.ResourceMemory: ParseQuantityOrDie("100Mi"),
 		}
-		klog.Infof("d: %+v", d)
+		klog.InfoS("d", "d", d)
```
Comment (kwiesmueller) [inline vertical-pod-autoscaler/e2e/v1/admission_controller.go:160]: Let's find out what this line is meant for and give it a more useful text. I'm not entirely sure this line should actually be there, I thought tests should log with something like t.Log() instead anyways.

File: vertical-pod-autoscaler/e2e/v1/admission_controller.go [hunk 2]
```diff
@@ -286,7 +286,7 @@ var _ = AdmissionControllerE2eDescribe("Admission-controller", func() {
 			apiv1.ResourceCPU:    ParseQuantityOrDie("400m"),
 			apiv1.ResourceMemory: ParseQuantityOrDie("600Mi"),
 		}
-		klog.Infof("d: %+v", d)
+		klog.InfoS("d", "d", d)
```
Comment (kwiesmueller) [inline vertical-pod-autoscaler/e2e/v1/admission_controller.go:289]: Same here

**Review by kwiesmueller — CHANGES_REQUESTED** [review]

**Comment by adrianmoisey** [comment]
@omerap12 can you address the remaining comments here?

**Comment by omerap12** [comment]
> @omerap12 can you address the remaining comments here?

Of course, I will adjust the code later

File: vertical-pod-autoscaler/e2e/v1/admission_controller.go (hunk 1 above)
Comment (omerap12) [reply vertical-pod-autoscaler/e2e/v1/admission_controller.go]: Good catch. I've updated the code to use `framework.Logf`, similar to how it's implemented here: https://github.com/kubernetes/kubernetes/blob/master/test/e2e/apps/replica_set.go#L643.

File: vertical-pod-autoscaler/e2e/v1/admission_controller.go [hunk 3]
```diff
@@ -157,7 +156,7 @@ var _ = AdmissionControllerE2eDescribe("Admission-controller", func() {
 			apiv1.ResourceCPU:    ParseQuantityOrDie("100m"),
 			apiv1.ResourceMemory: ParseQuantityOrDie("100Mi"),
 		}
-		klog.Infof("d: %+v", d)
+		framework.Logf("Created Hamster deployment %v", d)
```
Comment (adrianmoisey) [inline vertical-pod-autoscaler/e2e/v1/admission_controller.go:159]:
```suggestion
		framework.Logf("Created hamster deployment %v", d)
```

Just to be consistent with the other log messages above

File: vertical-pod-autoscaler/e2e/v1/admission_controller.go [hunk 4]
```diff
@@ -286,7 +285,7 @@ var _ = AdmissionControllerE2eDescribe("Admission-controller", func() {
 			apiv1.ResourceCPU:    ParseQuantityOrDie("400m"),
 			apiv1.ResourceMemory: ParseQuantityOrDie("600Mi"),
 		}
-		klog.Infof("d: %+v", d)
+		framework.Logf("Created Hamster deployment %v", d)
```
Comment (adrianmoisey) [inline vertical-pod-autoscaler/e2e/v1/admission_controller.go:288]:
```suggestion
		framework.Logf("Created hamster deployment %v", d)
```

Just to be consistent with the other log messages above

Comment (omerap12) [reply vertical-pod-autoscaler/e2e/v1/admission_controller.go:288]: sure

**Comment by voelzmo** [comment]
Hey @omerap12, thanks for the PR. AFAIK, we only use `klog` in the e2e tests whenever the `framework` couldn't be created due to some error – in all other cases we use the `framework.LogX` methods. I guess we didn't catch this properly in the previous PR, sorry about that.
Could you please adapt the other usages in the e2e tests which you're touching here to use the already existing logger in `framework`?
Thanks!

**Pull request merged**